lib/teaspoon/configuration.rb

```diff
@@ -21,7 +21,8 @@ class Configuration
     @@asset_paths    = ["spec/javascripts", "spec/javascripts/stylesheets",
                         "test/javascripts", "test/javascripts/stylesheets"]
     @@fixture_paths  = ["spec/javascripts/fixtures", "test/javascripts/fixtures"]
-    @@asset_manifest = ["teaspoon.css", "teaspoon-filterer.js", "teaspoon/*.js", "support/*.js"]
+    @@asset_manifest = ["teaspoon.css", "teaspoon-filterer.js", "teaspoon-manifest.js"] +
```

Comment:
This used to be this way because you could include the teaspoon.coffee files -- if you had coffeescript available, and wanted to debug it -- but this was generally a development tool. I think it may still be referenced in the generator or something so this is a note to myself to clean that incorrect information up.